.travis.yml

```diff
@@ -0,0 +1,22 @@
+language: node_js
```

Comment:
@nherment I think something went wrong with the rebase? This file is already in master :/